posts/ethereum-development-tips.md

```diff
@@ -0,0 +1,254 @@
+---
+authors:
+- Louis Fournier
+- Hugo de Almeida Rodrigues
+tags:
+- Blockchain
+- Ethereum
+- Dai
+- Stablecoin
+- Solidity
+- Truffle
+- Remix
+- ZeppelinOS
+- web3
+- Metamask
+- geth
+date: 
+title: "Ethereum Development Tips"
+image: 
+---
+
+{{< responsive-figure src="/images/2018/08/blockchain-interns-01.png" >}}
+(source: https://coin24.fr/2018/03/07/faut-vraiment-acheter-ethereum-2018/)
+
+# Summary
```

Comment:
there will already be a title level 1, so this can be removed to make `When we arrived...` the first text block